modules/test-utils/src/browser-automation/browser-test-driver.ts

```diff
@@ -56,17 +71,20 @@ export default class BrowserTestDriver extends BrowserDriver {
       color: COLOR.BRIGHT_YELLOW
     })();
 
-    // Backward compatibility: if `server` is not defined, fallback to config object
-    return this._startServer(config.server || config)
-      .then(url => {
-        return this._openPage(url, config);
-      })
-      .then(result => {
-        return this._onFinish(result);
-      })
-      .catch(error => {
-        this._fail(error.message || error);
-      });
+    try {
+      // TODO - Backward compatibility: if `server` is not defined, fallback to config object
+      const url = await this._startServer(config.server || config);
```

Comment:
Probably don't need this fallback any more in new major release

Reply:
Yeah, I tried to simplify it but ran into some weird stuff going on in _startServer, it accepts a function in place of config object etc. The typings don't really seem to work for these cases, it doesn't seem to be plumbed through. I can take it out but let me do it in a separate PR so that you can see what is going on.